content/20.results.md

```diff
@@ -102,14 +102,14 @@ Estimated composition of name origin prediction over the years of
 ### Affiliation Analysis
 
 We analyzed the countries of affiliation between corresponding authors and ISCB honorees.
-For each country, we report a value of log enrichment (LOE) and its 95% confidence intervals  (Table @tbl:country-enrichment).
+For each country, we report a value of log~2~ enrichment (LOE) and its 95% confidence intervals (Table @tbl:country-enrichment).
 A positive value of LOE indicates a higher proportion of honorees affiliated with that country compared to authors.
 A LOE value of 1 represents a one-fold enrichment (i.e., observed number of honorees is twice as much as expected).
 In the 20 countries with the most publications, we found an overrepresentation of honorees affiliated with institutions and companies in the US (97 speakers more than expected, LOE = 0.6, 95% CI (0.5, 0.8)) and Israel (12 speakers more than expected, LOR = 1.6, 95% CI (0.9, 2.3)), and an underrepresentation of honorees affiliated with those in China, France, Italy, the Netherlands, Taiwan, and India (Fig. @fig:country-enrichment).
 
-![The overrepresentation of honorees affiliated with institutions and companies in the US and Israel contrasts the underrepresentation of honorees affiliated with those in China, France, Italy, the Netherlands, Taiwan, and India. 
-For each country, enrichment is computed by dividing the observed proportion of honorees by the expected proportion of honorees whose affiliations are in that country, and 95% confidence interval of the log is estimated with the Poisson model method (left). 
-Observed (triangle) and expected (circle) number of honorees and their differences (observed - expected) are shown in square-root scale on the right. 
+![The overrepresentation of honorees affiliated with institutions and companies in the US and Israel contrasts the underrepresentation of honorees affiliated with those in China, France, Italy, the Netherlands, Taiwan, and India.
+Each country's log~2~ enrichment (LOE) and its 95% confidence interval are displayed on the left.
+Observed (triangle) and expected (circle) number of honorees and their differences (observed - expected) are shown in square-root scale on the right.
 Countries are ordered based on the proportion of authors in the field.
 
 ](https://raw.githubusercontent.com/greenelab/iscb-diversity/a61b73ed80ba2062d31a1f111c1b8f388fda4b3a/figs/enrichment-plot.png){#fig:country-enrichment width="80%"}
```

Comment:
Does this commit hash need to change?

Reply:
Hmm... it's already changed, hasn't it?

![hash-change](https://user-images.githubusercontent.com/1679452/78150114-d296a280-7404-11ea-9c96-6c485669a620.png)